### PR TITLE
ROX-9450: Removing classic policies code

### PR DESCRIPTION
**Pull request by alwayshooin**
## Description

As the title suggests. Removed dead code in Violations detail page as well for the old way of showing policy details. Sorry for the big PR -- it's all deletes and moves, I swear!

## Checklist
- [ ] Investigated and inspected CI test results
- [ ] Unit test and regression tests added
- [ ] Evaluated and added CHANGELOG entry if required
- [ ] Determined and documented upgrade steps
- [ ] Documented user facing changes (create PR based on [stackrox/openshift-docs](https://github.com/stackrox/openshift-docs) and merge into [rhacs-docs](https://github.com/openshift/openshift-docs/tree/rhacs-docs))

If any of these don't apply, please comment below.

## Testing Performed

TODO(replace-me)
Use this space to explain how you tested your PR, or, if you didn't test it, why
you did not do so. Valid reasons include, for example, "CI is sufficient",
"No testable changes". Feel free to attach JSON snippets, curl commands,
screenshots.

In addition to reviewing your code, reviewers **must** also review your testing
instructions and make sure they are sufficient.

### CODE REVIEW

File: ui/apps/platform/src/reducers/policies/table.js
Comment (alwayshooin) [inline ui/apps/platform/src/reducers/policies/table.js:5]: it looks like update policy disabled state is being used in vuln mgmt so i left it as is for now.

Comment (vjwilson) [reply ui/apps/platform/src/reducers/policies/table.js:5]: The e2e test that's failing in CI looks like it may be related to this, so maybe there's something else we need to retain from the old code.

**Review by vjwilson — COMMENTED** [review]
Looks good to me, but I will defer to Mark for a better set of eyes on it.

(Also, need to fix test that's failing in CI.)

**Comment by dvail** [comment]
The Dashboard Search/URL cleanup I'm doing leads down a long path that ends with a reducer that is only used in this dead code.

Commenting just to keep an eye out so I can rebase and delete more once this is merged.

**Comment by pedrottimark** [comment]
@alwayshooin failing integration test borrows a selector for classic policies wizard, but behavior is still correct

https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/cypress/integration/vulnmanagement/policiesListpages.test.js#L40-L42

Classic page before removal
policiesListpages-classic

PatternFly page after removal
policiesListpages

File: ui/apps/platform/src/Containers/Violations/Details/ViolationDetailsPage.tsx
Comment (pedrottimark) [inline ui/apps/platform/src/Containers/Violations/Details/ViolationDetailsPage.tsx:16]: How about absolute `Containers/Policies` instead of relative `../../Policies` across containers?

File: ui/apps/platform/src/reducers/policies/table.js
Comment (pedrottimark) [reply ui/apps/platform/src/reducers/policies/table.js:5]: Failing integration test is because of classic selector for PatternFly criteria element

**Review by pedrottimark — APPROVED** [review]
Wow, super!
* Wrote hypothesis about failing integration test.
* Made 2 suggestions about import paths.
* Rebase merge conflicts with #1325

@fredrb you will need to rebase #1194 after this has been merged

**Comment by pedrottimark** [comment]
@alwayshooin It looks like a leftover absolute path to PatternFly folder might become relative path to new location:

Failed to compile.
src/Containers/Policies/Table/PoliciesTable.tsx
Cannot find module: 'Containers/Policies/PatternFly/Modal/EnableDisableNotificationModal'.

**Pull request merged**
